hackclub.community.yaml

```
@@ -16,4 +16,8 @@ _atproto.alumni:
 _github-pages-challenge-hackclub-alumni.alumni:
   - ttl: 600
     type: TXT
-    value: "62688903c020be82f48ace03d54467"
\ No newline at end of file
+    value: "62688903c020be82f48ace03d54467"
+blog.alumni:
+  - ttl: 600
+    type: CNAME
+    value: "731fb7077db63b1d.vercel-dns-016.com."
```

Remove quotes. Technically might be valid with them, but it's a little weird
```suggestion
    value: 731fb7077db63b1d.vercel-dns-016.com.
```

---

Noted, amending soon.

(For those wandering why I `git commit --amend`, it's a practice in Alpine Linux to do that when updating your patch to [their ports repo](https://gitlab.com/alpine/aports) as you follow through maintainer reviews.)